_data/glossary/cluster-architect.yaml

```diff
@@ -0,0 +1,12 @@
+id: cluster-architect
+name: Cluster Architect
+related:
+  - cluster
+  - cluster-operator
+tags:
+  - user-type
+short-description: |
+  A person who designs infrastructure that involves one or more Kubernetes clusters.
+
+long-description: |
+  They are concerned with distributed system best practices, such as high availability and security.
```

Comment:
Cluster architects are concerned with best practices for distributed systems, for example: high availability and security.